### PR TITLE
Fixed the crash when there is no language style specific.

### PR DESCRIPTION
**Pull request by benevbright**
If I don't assign any language style, app crashes.
Especially when I use `allinc-plugin`, which loads all plugins.

### CODE REVIEW

File: plugins/prism/test.js [hunk 1]
```diff
@@ -29,4 +29,16 @@ describe("Prism plugin", () => {
       .to.have.selector("code span.token.punctuation")
       .with.text.to.equal("#")
   })
+
+  it("Don't highlight code without language specific", () => {
```
Comment (hagenburger) [inline plugins/prism/test.js]: Optional: If you want to best typography, write `Don’t` instead of `Don't` :)

**Comment by benevbright** [comment]
Good catch! I'll check and update it tomorrow. :)

**Comment by benevbright** [comment]
Unfortunately, <p> not get HTML-escaped. 
So I had to add more code.

File: plugins/prism/test.js [hunk 2]
```diff
@@ -29,4 +29,28 @@ describe("Prism plugin", () => {
       .to.have.selector("code span.token.punctuation")
       .with.text.to.equal("#")
   })
+
+  it("Don’t highlight code without language specific", () => {
```
Comment (hagenburger) [inline plugins/prism/test.js]: Can you rephrase this? `it` + the string should be read as one sentence. Sot the string should always start with “should”.

bildschirmfoto 2018-10-05 um 08 05 39

I noticed I made a mistake myself. I’ll fix my tests in a separate PR.

File: plugins/prism/test.js [hunk 3]
```diff
@@ -29,4 +29,28 @@ describe("Prism plugin", () => {
...
+    const input = unindent`
+      ~~~
+      <p>
+      ~~~
+    `
+    const doc = new Document(input)
+    doc.config.use(plugin)
+    const html = doc.render()
+    expect(html).to.not.have.selector("code span.token")
```
Comment (hagenburger) [inline plugins/prism/test.js]: Can you test for the escaping, please? E.g. match the `innerHTML`.

File: plugins/prism/test.js [hunk 4]
```diff
@@ -29,4 +29,28 @@ describe("Prism plugin", () => {
...
+  it("Don’t highlight code with unsupported language", () => {
+    const input = unindent`
+      ~~~superjs
+      <p>
+      ~~~
+    `
+    const doc = new Document(input)
+    doc.config.use(plugin)
+    const html = doc.render()
+    expect(html).to.not.have.selector("code span.token")
```
Comment (hagenburger) [inline plugins/prism/test.js]: Can you test for the escaping, please? E.g. match the `innerHTML`.

File: plugins/prism/test.js [hunk 5]
```diff
@@ -29,4 +29,28 @@ describe("Prism plugin", () => {
...
+  })
+
+  it("Don’t highlight code with unsupported language", () => {
```
Comment (hagenburger) [inline plugins/prism/test.js]: See comment above.

File: lib/renderer/example.js [hunk 6]
```diff
@@ -24,12 +24,15 @@ module.exports = class Example extends Fragment {
...
-        code = highlight(code, this.info.lang)
+      let highlightedCode = null
+      if (highlight) highlightedCode = highlight(code, this.info.lang)
+      if (highlightedCode !== null) {
+        code = highlightedCode
       } else {
         this.code.textContent = code
         code = this.code.innerHTML
       }
+
```
Comment (hagenburger) [inline lib/renderer/example.js]: Why this additional line?

**Comment by benevbright** [comment]
Thanks for your review. 👍
